rustls/src/lib.rs

```diff
@@ -59,11 +59,13 @@
 //! While Rustls itself is platform independent it uses
 //! [`ring`](https://crates.io/crates/ring) for implementing the cryptography in
 //! TLS. As a result, rustls only runs on platforms
-//! supported by `ring`. At the time of writing this means x86, x86-64, armv7, and
-//! aarch64. For more information see [the supported `ring` CI
-//! targets](https://github.com/briansmith/ring/blob/9cc0d45f4d8521f467bb3a621e74b1535e118188/.github/workflows/ci.yml#L151-L167).
+//! supported by `ring`. At the time of writing this means x86, x86-64, aarch64,
+//! armv7, powerpc64le, riscv64gc and s390x. For more information, see [the
+//! supported `ring` CI targets][ring-ci-targets].
```

Comment:
Ditto.

Reply:
The README content is pulled from `src/lib.rs` w/ [`admin/pull-readme`](https://github.com/rustls/rustls/blob/main/admin/pull-readme) and [checked in CI](https://github.com/rustls/rustls/blob/5427a4d6e99ffcb67ffa1adcb9d690e320c8de57/.github/workflows/build.yml#L179-L184) so we can fix this in one place and resolve both. I'll mark this thread resolved since the substantive feedback is in the other comment thread on `README.md`.